main.go

```diff
@@ -112,6 +180,13 @@ func main() {
 	}
 	logrus.SetLevel(level)
 
+	// Resolve connect URL if the connection scheme is tcp or udp
+	resolvedHost, err := resolveNsmConnectURL(ctx, &c.ConnectTo)
+	if err != nil {
+		logrus.Fatalf("error resolving nsm connect host: %v, err: %v", c.ConnectTo, err)
```

Comment:
Incase of error I see whole main.go continues to proceed further? Do we need to exit or proceed further

Reply:
@rajendra-avesha Fatalf logs the message and exits the process with a non-zero status so the program does not proceed after this point.